example_dot_env

```diff
@@ -1,4 +1,9 @@
 PORT=3030
+
+SESSION_SECRET=some_secret
```

Comment:
yeah this is just some secret key used to secure the session, required by `express-session`. i'll look up the requirements and add them to the readme.